README.md

```diff
@@ -0,0 +1,37 @@
+Super Chemex Bot
+================
+
+Our friendly bot-overlord notifying people when there is freshly brewed coffee in the kitchen.
+
+Super Chemex Bot is a Slack app that posts a message in channel when an AWS IoT button is pressed.
+
+Setup
+=====
+
+Super Chemex Bot is only a small piece of code but requires a lot of configuration to get running. Below is a high level overview of the steps you'll have to take to get it working.
+
+ * Clone this repository
+ * `cd` into the `lambda` directory and run `npm install` (`yarn install` is also supported). `cd` back into the parent directory
```

Comment:
why do i have to cd back in this step?

Reply:
hmm, fair point. I could change line 31 to use `../deploy.sh`. Though I'm not sure if that's confusing. WDYT?

Reply:
You could say "Navigate to the project root folder and run `./deploy.sh`"